src/app/list/list.component.ts

```diff
@@ -0,0 +1,71 @@
+import {  AfterViewChecked,  ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Observable, of } from 'rxjs';
+
+import { IEntity } from '../entity';
+import {  selectCurrentItems, selectMapEntitiesList } from '../store/coronaLocations.selector';
+import * as coronaLocationsActions from '../store/coronaLocations.action';
+import { IMapEntity } from '../mapEntity';
+import { ActionType } from 'angular-cesium';
+import { MapService } from '../map.service'
+@Component({
+  selector: 'app-list',
+  templateUrl: './list.component.html',
+  styleUrls: ['./list.component.css'],
+})
+
+export class ListComponent implements OnInit,AfterViewChecked {  
+  private _newMapEntitiesList: Observable<IEntity[]> = new Observable<IEntity[]>();
+  private _mapEntities: Observable<IMapEntity[]> = new Observable<IMapEntity[]>()
+  constructor(private _store: Store<{storeCoronaLocation: any}>,
+    private _mapService: MapService,
+    private _changeDetector: ChangeDetectorRef) { 
+  }
+  get newMapEntitiesList(): Observable<IEntity[]>{
+    return this._newMapEntitiesList;
+  }
+  get mapEntities(): Observable<IMapEntity[]>{
+    return this._mapEntities;
+  }
+  set newMapEntitiesList(newMapEntitiesList: Observable<IEntity[]>){
+    this._newMapEntitiesList=newMapEntitiesList;
+  }
+  set mapEntities(mapEntities: Observable<IMapEntity[]>){
+    this._mapEntities=mapEntities;
+  }
+  ngOnInit(): void {
+    this._store.select(selectCurrentItems).subscribe(data=> this.newMapEntitiesList=(of(data)));
+    this._store.select(selectMapEntitiesList).subscribe(data=>{
+      this.mapEntities=(of(data.filter(item=>item.entity.saved==true)))
+     })       
+  }
+  ngAfterViewChecked(): void{
+    this._changeDetector.detectChanges();
+  }
+
+  onClickCancel(entity: IEntity): void{
+    let entityTmp: IMapEntity = {
+      id:entity.id,
+      entity:{
+        position: entity.position,
+        id: entity.id,
+        saved: false
+      },
+      actionType: ActionType.ADD_UPDATE 
```

Comment:
why action update? also, why do you sent the entire entity as payload and not just it's id?